test/functorch/test_control_flow.py

```diff
@@ -1624,6 +1624,142 @@ def test_cond_tracing_with_valid_inputs(self, predType, innerFnType, nOperands,
                 gm = make_fx(fn, tracing_mode=tracing_mode, _allow_non_fake_inputs=True)(*args)
                 self.assertEqual(gm(*args), eager_res)
 
+    @parametrize("predType", ["boolTensor"])
+    @parametrize("innerFnType", ["function", "module", "object"])
+    @parametrize("nOperands", [1, 2])
+    @parametrize("nClosure", [0, 1])
+    @parametrize("nesting", [0])
+    def test_cond_vmap(self, predType, innerFnType, nOperands, nClosure, nesting):
```

Comment:
Btw, before we merge this PR, we should fix our testing. We have a CI configuration that runs all of the functorch tests with Dynamo turned on, but for reasons that CI configuration doesn't actually run the functorch tests.

Reply:
I'll submit a couple of PRs for this

Reply:
Stack over at https://github.com/pytorch/pytorch/pull/115267